Formula/ecflow-ui.rb

```diff
@@ -0,0 +1,40 @@
+class EcflowUi < Formula
+  desc "User interface for client/server workflow package"
+  homepage "https://confluence.ecmwf.int/display/ECFLOW"
+  url "https://confluence.ecmwf.int/download/attachments/8650755/ecFlow-5.8.1-Source.tar.gz"
+  sha256 "be2ff0b76da2917a00ba13cd02b2992616e374b102ea92dff63a022fbdaa1884"
+  license "Apache-2.0"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
```

Comment:
Doesn't it work with the llvm in macOS?

Reply:
Hi @SMillerDev yes it does, this is my first brew formula and I copied some bits from another one. I guess I can just remove gcc from the requirements - I'll try it and update the PR.